### ISSUE TITLE
SD init failed! when attempting to launch any game

### ISSUE THREAD

**Issue by npollender** [Robz8/TWLoader#452]
### Your 3DS

Tell us which console do you have:

- [ ]  Old 3DS
- [x]  Old 3DS XL
- [ ]  2DS
- [ ]  New 3DS
- [ ]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

- [ ]  Boot9Strap (SigHax)
- [x]  A9LH (Only Sysnand)
- [ ]  A9LH + Emunand
- [x]  Luma3DS
- [ ]  AuReiNAND
- [ ]  Other CFW [specify]
- [ ]  Older CFW
- [ ]  Not using CFW / Using TWLoader in 3DSX format

### Console version

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### TWLoader version

Tell us which versions you are using:

- [x]  Latest release
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release

### Expected behaviour

Bootstrap loading screen should appear and game launch afterwards.

### Actual behaviour

Top screen remains white and bottom screen goes black with white text written “SD init failed!”

### Steps to reproduce

1. Launch any game (donor rom changes nothing)
2. Stuck on init failed screen

### Stack trace and console log

If you are getting a crash or want to upload your log file, please, write that information between the following quotation marks.

```
```

**Comment by npollender** [comment]
Bump?

**Comment by Robz8** [comment]
Is it still happening with nds-bootstrap v0.7.4?

**Comment by npollender** [comment]
Yes, I’m afraid. The message only appears for a fraction of a second now. The top screen remains white and the bottom screen remains black indefinitely.

**Comment by Robz8** [comment]
Try reformatting your SD card.

**Comment by npollender** [comment]
I have several times.

**Comment by npollender** [comment]
It’s a 32GB Lexar Platinum II SDHC

**Comment by Robz8** [comment]
It's possible nds-bootstrap may be incorrectly recognizing the SD card.
Try blowing the slot, and then reinserting it.

**Comment by npollender** [comment]
Still doesn't work.

**Comment by Robz8** [comment]
Does a previous version of bootstrap work?

**Comment by npollender** [comment]
No, it's been a problem for several months.
